recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "statannotations" %}
-{% set version = "0.6.0" %}
+{% set version = "0.7.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/statannotations-{{ version }}.tar.gz
```

Isn't it this we should update?
```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/statannotations-{{ version }}.tar.gz
```